src/uct/ib/rc/base/rc_ep.h

```diff
@@ -185,7 +193,7 @@ static inline void uct_rc_ep_process_tx_completion(uct_rc_iface_t *iface,
 
     ucs_queue_for_each_extract(op, &ep->outstanding, queue,
                                UCS_CIRCULAR_COMPARE16(op->sn, <=, sn)) {
-        op->handler(op);
+        op->handler(op, UCS_OK);
```

Comment:
It is done during ep desctruction, which is invoked by uct_ep_set_failed. 
It is called with ERR_CANCELLED status